package.json

```diff
@@ -13,7 +14,7 @@
         "@wdio/local-runner": "^8.11.2",
         "@wdio/mocha-framework": "^8.11.0",
         "@wdio/spec-reporter": "^8.11.2",
-        "chromedriver": "^114.0.2",
+        "chromedriver": "^116.0.0",
```

Comment:
Just in case your tests will stop working. Check our chat I've added changes to the repo as WDIO released a new version and everything becomes a little easier, but it will require some small changes in your repo. Feel free to chat me for more info